packaging/debian-sid/patches/0001-cmd-snap-seccomp-use-upstream-seccomp-package.patch

```diff
@@ -21,11 +21,11 @@ Signed-off-by: Maciej Borzecki <maciej.zenon.borzecki@canonical.com>
  cmd/snap-seccomp/versioninfo_test.go | 2 +-
  4 files changed, 4 insertions(+), 7 deletions(-)
 
-diff --git a/cmd/snap-seccomp/main.go b/cmd/snap-seccomp/main.go
```

Comment:
Why all the .patch changes?

Reply:
Our debian-quilt patch touches cmd/snap-seccomp/versioninfo.go which is also modified by this branch. This leads to a "fuzzy" apply of the patch. Unfortunately dpkg-buildpackage complains when there is fuzziness in the diff so it needed to be refreshed.